dmscripts/notify_buyers_when_requirements_close.py

```diff
@@ -49,6 +49,9 @@ def notify_users(email_api_key, stage, brief):
                 extra={'error': e, 'brief_id': brief['id']}
             )
 
+            if isinstance(e, EmailTemplateError):
```

Comment:
When will this happen? Looking in dmutils, it doesn't look as if `EmailTemplateError` is a subclass of `EmailError`?

Reply:
Oh farts. That was a mistake! Thanks for spotting. I think will update utils to reflect my original intention.

Reply:
Fixed in alphagov/digitalmarketplace-utils#588